org.eclipse.scout.rt.platform.test/src/main/java/org/eclipse/scout/rt/testing/platform/runner/RunWithNewPlatform.java

```diff
@@ -16,8 +16,8 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import org.eclipse.scout.rt.platform.DefaultPlatform;
 import org.eclipse.scout.rt.platform.IPlatform;
+import org.eclipse.scout.rt.testing.platform.TestingDefaultPlatform;
 
 /**
  * Annotation to execute test-methods using a new {@link IPlatform}, not the globally shared Platform. This annotation
```

Comment:
Add a note, indicating that a custom implemented platform used together with this annotation should use TestingPlatform as base class.

Reply:
Agreed, added an additional paragraph to the annotation.